src/components/template/PdfTemplatesTable.tsx

```diff
@@ -0,0 +1,35 @@
+import { Column } from '@material-table/core';
+import React from 'react';
+
+import { Template, TemplateGroupId } from 'generated/sdk';
+
+import TemplatesTable, { TemplateRowDataType } from './TemplatesTable';
+
+type PdfTemplatesTableProps = {
+  dataProvider: () => Promise<
+    Pick<
+      Template,
+      'templateId' | 'name' | 'description' | 'isArchived' | 'questionaryCount'
+    >[]
+  >;
+};
+
+const columns: Column<TemplateRowDataType>[] = [
+  { title: 'Name', field: 'name' },
+  { title: 'Description', field: 'description' },
+];
+
+function PdfTemplatesTable(props: PdfTemplatesTableProps) {
```

Comment:
I'm not that good at React, but does this component ever get used? Seems like the sidebar link goes to `PdfTemplatesPage`, which includes `DefaultTemplatesTable` rather than this table

Reply:
Thanks, it was unused! I made it when I thought I was going to have to create my own and forgot to remove it :)